src/Components/InputCard/inputcard.css

```diff
@@ -1,48 +1,49 @@
-.container-input-new-note{
-    height: fit-content;
-    width: 40rem;
-    position: relative;
-    border: transparent;
-    border-radius: 0.5rem;
+.container-input-new-note {
+  height: fit-content;
+  width: 40rem;
+  position: relative;
+  border: transparent;
+  border-radius: 0.5rem;
 }
 
-.title-new-note, .text-new-note {
-    width: 100%;
-    background: transparent;
-    margin: 0;
-    border: none;
-    padding-right: 2rem;
-    outline: none;
-    word-wrap: break-word;
-    resize: none;
+.title-new-note,
+.text-new-note {
+  width: 100%;
+  background: transparent;
+  margin: 0;
```

Comment:
suggestion: use this property in the root directory

Reply:
Yes will add it. Thanks for pointing this out